### ISSUE TITLE
[BUG] Predictive text cannot be disabled

### ISSUE THREAD

**Issue by jsfan3** [codenameone/CodenameOne#2905]
Test case:
```
Form hi = new Form("Hi World", BoxLayout.y());
        hi.add(new TextField("", "Type here (TextArea.ANY)", 80, TextArea.ANY));
        hi.add(new TextField("", "Type here (TextArea.SENSITIVE)", 80, TextArea.SENSITIVE));
        hi.add(new TextField("", "Type here (TextArea.NON_PREDICTIVE)", 80, TextArea.NON_PREDICTIVE));
        hi.add(new TextField("", "Type here (TextArea.UPPERCASE)", 80, TextArea.UPPERCASE));
        hi.show();
```

Result: Android ignores `TextArea.SENSITIVE` and `TextArea.NON_PREDICTIVE`. This is a new issue, because in the past there is no prediction at all. About iOS, my iPhone never shows predictive text, so I don't know how to test it.

**Comment by shannah** [comment]
I don't think this was a regression.  I can't find any evidence that SENSITIVE ever was used on the Android port.  Similarly, no changes were made recently related to NON_PREDICTIVE in the android port.  It is more likely that this is a variation between Android distributions, versions, and devices, as described [here](https://stackoverflow.com/questions/3789905/how-can-i-turn-off-suggestions-in-edittext).

I have now added support for SENSITIVE, and I have added the `InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD` flag in both cases which seems to ensure that no suggestions are shown when typing.
